### PR TITLE
test: improve test names

### PR DESCRIPTION
**Pull request by kblok**
We have a few tests with duplicated names and some describes with bad names

### CODE REVIEW

File: test/page-emulate-media.spec.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ it('should emulate type', async ({page, server}) => {
   expect(await page.evaluate(() => matchMedia('print').matches)).toBe(false);
 });
 
-it('should throw in case of bad type argument', async ({page, server}) => {
+it('should throw in case of media argument', async ({page, server}) => {
```
Comment (yury-s) [inline test/page-emulate-media.spec.ts:35]: nit: ...bad media...

File: test/popup.spec.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ it('should use viewport size from window features', async function({browser, ser
   expect(resized).toEqual({width: 500, height: 400});
 });
 
-it('should respect routes from browser context', async function({browser, server}) {
+it('should respect routes from browser context using window.open', async function({browser, server}) {
```
Comment (yury-s) [inline test/popup.spec.ts:149]: nit: ...when using window.open ?

**Review by yury-s — APPROVED** [review]

File: test/page-emulate-media.spec.ts (hunk 1 above)
Comment (kblok) [reply test/page-emulate-media.spec.ts]: done

File: test/popup.spec.ts (hunk 2 above)
Comment (kblok) [reply test/popup.spec.ts]: done

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**